conf/default.toml

```diff
@@ -14,7 +14,7 @@ proxy_path = ""
 # Use ISO 639-2/B (https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) format to indicate the language of the metadata.
 # Setting to "current" will use the current language of the User Interface.
 # If not indicated, a wildcard is used and no language preference is applied for the search.
-# metadata_language = "current"
+metadata_language = "current"
```

Comment:
I would leave this commented; even though it's a logical behaviour for multilingual catalogs I think the default conf should only contain mandatory keys